src/assets/content/faqContent.tsx

```diff
@@ -10,19 +11,21 @@ const LinkDestinations = {
   turboTaxGuarantees: "https://turbotax.intuit.com/corp/guarantees/",
 } as const;
 
-const Link: React.FC<{ to: keyof typeof LinkDestinations }> = (props) => (
-  <BaseLink
+const Link: React.FC<{ to: keyof typeof LinkDestinations }> = (props) => {;
```

Comment:
Is this a valid syntax? With the ; after the {

Reply:
No, let me remove it.